python/llm/test/inference/test_optimize_model.py

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pytest
+import os
+
+from bigdl.llm.transformers import AutoModelForCausalLM, AutoModel
+from transformers import LlamaTokenizer, BloomTokenizerFast, GPTNeoXTokenizerFast, AutoTokenizer
```

Comment:
Can we all use AutoTokenizer and AutoModelForCausalLM?

Reply:
llama-7b-hf requires LlamaTokenizer. Using AutoTokenizer throws RecursionError: maximum recursion depth exceeded while getting the str of an object

chatglm2-6b requires AutoModel. Using AutoModelForCausalLM throws ValueError: Unrecognized configuration class <class 'transformers_modules.chatglm2-6b.configuration_chatglm.ChatGLMConfig'> for this kind of AutoModel: AutoModelForCausalLM.

bloomz-7b1 can work with AutoTokenizer instead of BloomTokenizerFast.

Reply:
Sure, then we can keep LlamaTokenizer and chatglm AutoModel.